setup.cfg

```diff
@@ -9,6 +9,6 @@ requires-dist =
         docutils>=0.10,<0.15
         rsa>=3.1.2,<=3.5.0
         PyYAML>=3.10,<=3.13; python_version=="2.6"
-        PyYAML>=3.10,<=5.1;python_version!="2.6"
+        PyYAML>=3.10,<=5.1.1;python_version!="2.6"
```

Comment:
I think `<5.2` is better. Maybe bump major version when backward incompatible changes.